src/tests/Interop/DllImportSearchPaths/DllImportSearchPathsTest.cs

```diff
@@ -39,6 +39,15 @@ public static void AssemblyDirectory_Found()
         Assert.Equal(3, sum);
     }
 
+    public static bool IsNativeAot => TestLibrary.Utilities.IsNativeAot;
+
+    [ConditionalFact(nameof(IsNativeAot))]
```

Comment:
You could do `[ConditionalFact(typeof(TestLibrary.Utilities), nameof(TestLibrary.Utilities.IsNativeAot))]` instead of defining a separate property on this class.